### ISSUE TITLE
problems with freemix

### ISSUE THREAD

**Issue by williajgr** [FrSkyRC/ETHOS-Feedback-Community#4007]
Ok i had a freemix set up for my flaps which worked perfectly.  Today i was trying to change the mix and couldn't figure out how to change it.  I finally figured out that one of the updates turned it into a vars which i have never used before making a simple flaps setup very complicated.   sSo i tried to redo it in the freemix and don't understand what the heck it is doing. If you look at the picture one can see that when se is up it should be positive 44 and it is negative 44. it seems like ethos continues to be messed up.  verry irritating.
![flaps](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/56521075/548b4333-bcfd-4496-8d2d-99d53a41e22f)

**Comment by Chilie78** [comment]
I'm not sure what mix you were trying to do there, but I think you're either using the wrong one or making this more complicated. Use a simple free mix and set the weight up and down down at the bottom where you can see the read out for each servo. The way you have it won't give you the control over the servos like you probably want. This isn't a bug as I see it and more of a different way to accomplish a goal.
![20240509_162953](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/169282804/61f7eb23-2b85-46ed-8e4c-1517da536aad)

**Comment by T16-Pilot** [comment]
He is talking about "FREEMIX" you are showing a "FLAPS MIX"

The Freemix is now way more complicated than before 1.5.X....

In short almost every thing is now more complicated than before 1.5.X

I have been a Ethos user for almost 3 years now, and up until 1.4.17 i loved it. From 1.5.X i have started to thinking about leaving the system. 

A lot of us was using OTX back in the day because that was more or less the only option for a radio system that did not cost a lot and could do "every thing". But it was complicated to use for most people. There are those that like tinkering and programming. But most of us are people that want to fly and not spend a lot of time learning the radio system.

What i am trying to say, Ethos is getting more and more like OTX 2.0 and there are a lot of us that does not like the complexity that we are seeing in Ethos 1.5.X

I think that if Ethos continues in this direction people will leave the system because is is getting to a point where it is getting to advanced for most people.

I know the geeks love it, but they are not the major part of the users of this system, but they are maybe the loudest in regards to telling the Ethos crew what they want. The Ethos crew is trying to keep every body happy, and in that process Ethos is getting too complicated for a lot of "normal" sunday flyers...

Don´t get me wrong, i trust the system. And there is a lot of nice functions. It is just getting to a point where the software is getting to complicated for most people.

I am a nerd, and i like tinkering. Just not with the radio software. That needs to be simple and work every time.

For the record i use a X20 Pro, and half of the active pilots in my club are flying Ethos radios because i introduced them to the system.

Happy flying
Simon

**Comment by williajgr** [comment]
The flap mix is really stupid.  the three points (flaps up take off and landing) are never symmetrical and the left and right flaps need to have there own numbers. which is how it was before 1.5.0.  when the went to 1.5.0 they converted my freemix to having vars do all the work.  i didn't figure that out until i had to changer the flap mix.  I never used vars to do anything before.  now if i want to use the flap mix i have to use curves to get what i want. how about fixing it so i can just put 3 values in like the free mix was.!!!! guess the developers never had a plane with flaps

**Comment by williajgr** [comment]
the developers are going crazy changing things that shouldn't be changed. Making things too complicated for a lot of people. I happen to be an engineer that designs main frame computers and i don't like what you are doing. I have been in this hobby since 1965 and was a long time open tx user.  every time i look at an old model and try to make changes i get frustrated trying to figure out what was done to my model.  guess it may be time to switch to Jeti.  I have been selling Frsky as an alternative to there transmitters for a while.

**Comment by williajgr** [comment]
I just noticed that it changed my flap throws on many of my models.   Probably to the point that it will burn out the servos.  everything is +100    0     -100

**Comment by bsongis** [comment]
@williajgr would you please attach to the issue your model from 1.4.X so that I can check if the conversion is faulty or not?

**Comment by bsongis** [comment]
> The flap mix is really stupid. the three points (flaps up take off and landing) are never symmetrical and the left and right flaps need to have there own numbers. which is how it was before 1.5.0. when the went to 1.5.0 they converted my freemix to having vars do all the work. i didn't figure that out until i had to changer the flap mix. I never used vars to do anything before. now if i want to use the flap mix i have to use curves to get what i want. how about fixing it so i can just put 3 values in like the free mix was.!!!! guess the developers never had a plane with flaps

This is plain wrong:
1) The main developer has many planes and gliders :)
2) The freemix can have 3 values very easily, just use a weight (only one action) with 3 rates

**Comment by bsongis-frsky** [comment]
> He is talking about "FREEMIX" you are showing a "FLAPS MIX"
> 
> The Freemix is now way more complicated than before 1.5.X....
> 
> In short almost every thing is now more complicated than before 1.5.X
> 
> I have been a Ethos user for almost 3 years now, and up until 1.4.17 i loved it. From 1.5.X i have started to thinking about leaving the system.
> 
> A lot of us was using OTX back in the day because that was more or less the only option for a radio system that did not cost a lot and could do "every thing". But it was complicated to use for most people. There are those that like tinkering and programming. But most of us are people that want to fly and not spend a lot of time learning the radio system.
> 
> What i am trying to say, Ethos is getting more and more like OTX 2.0 and there are a lot of us that does not like the complexity that we are seeing in Ethos 1.5.X
> 
> I think that if Ethos continues in this direction people will leave the system because is is getting to a point where it is getting to advanced for most people.
> 
> I know the geeks love it, but they are not the major part of the users of this system, but they are maybe the loudest in regards to telling the Ethos crew what they want. The Ethos crew is trying to keep every body happy, and in that process Ethos is getting too complicated for a lot of "normal" sunday flyers...
> 
> Don´t get me wrong, i trust the system. And there is a lot of nice functions. It is just getting to a point where the software is getting to complicated for most people.
> 
> I am a nerd, and i like tinkering. Just not with the radio software. That needs to be simple and work every time.
> 
> For the record i use a X20 Pro, and half of the active pilots in my club are flying Ethos radios because i introduced them to the system.
> 
> Happy flying Simon

Which are the new features which cause you problems in 1.5.X?

**Comment by williajgr** [comment]
No one cannot use 3 weights. You have to use 2 weights and one offset.Sent from my iPhoneOn May 12, 2024, at 4:26 AM, Bertrand Songis @ FrSky ***@***.***> wrote:﻿

He is talking about "FREEMIX" you are showing a "FLAPS MIX"
The Freemix is now way more complicated than before 1.5.X....
In short almost every thing is now more complicated than before 1.5.X
I have been a Ethos user for almost 3 years now, and up until 1.4.17 i loved it. From 1.5.X i have started to thinking about leaving the system.
A lot of us was using OTX back in the day because that was more or less the only option for a radio system that did not cost a lot and could do "every thing". But it was complicated to use for most people. There are those that like tinkering and programming. But most of us are people that want to fly and not spend a lot of time learning the radio system.
What i am trying to say, Ethos is getting more and more like OTX 2.0 and there are a lot of us that does not like the complexity that we are seeing in Ethos 1.5.X
I think that if Ethos continues in this direction people will leave the system because is is getting to a point where it is getting to advanced for most people.
I know the geeks love it, but they are not the major part of the users of this system, but they are maybe the loudest in regards to telling the Ethos crew what they want. The Ethos crew is trying to keep every body happy, and in that process Ethos is getting too complicated for a lot of "normal" sunday flyers...
Don´t get me wrong, i trust the system. And there is a lot of nice functions. It is just getting to a point where the software is getting to complicated for most people.
I am a nerd, and i like tinkering. Just not with the radio software. That needs to be simple and work every time.
For the record i use a X20 Pro, and half of the active pilots in my club are flying Ethos radios because i introduced them to the system.
Happy flying Simon

Which are the new features which cause you problems in 1.5.X?

**Comment by williajgr** [comment]
If it was that easy why did the conversion change everything to vars?  Which it took me awhile to figure out what happened.Sent from my iPhoneOn May 12, 2024, at 5:42 AM, George Williams ***@***.***> wrote:﻿No one cannot use 3 weights. You have to use 2 weights and one offset.Sent from my iPhoneOn May 12, 2024, at 4:26 AM, Bertrand Songis @ FrSky ***@***.***> wrote:﻿

He is talking about "FREEMIX" you are showing a "FLAPS MIX"
The Freemix is now way more complicated than before 1.5.X....
In short almost every thing is now more complicated than before 1.5.X
I have been a Ethos user for almost 3 years now, and up until 1.4.17 i loved it. From 1.5.X i have started to thinking about leaving the system.
A lot of us was using OTX back in the day because that was more or less the only option for a radio system that did not cost a lot and could do "every thing". But it was complicated to use for most people. There are those that like tinkering and programming. But most of us are people that want to fly and not spend a lot of time learning the radio system.
What i am trying to say, Ethos is getting more and more like OTX 2.0 and there are a lot of us that does not like the complexity that we are seeing in Ethos 1.5.X
I think that if Ethos continues in this direction people will leave the system because is is getting to a point where it is getting to advanced for most people.
I know the geeks love it, but they are not the major part of the users of this system, but they are maybe the loudest in regards to telling the Ethos crew what they want. The Ethos crew is trying to keep every body happy, and in that process Ethos is getting too complicated for a lot of "normal" sunday flyers...
Don´t get me wrong, i trust the system. And there is a lot of nice functions. It is just getting to a point where the software is getting to complicated for most people.
I am a nerd, and i like tinkering. Just not with the radio software. That needs to be simple and work every time.
For the record i use a X20 Pro, and half of the active pilots in my club are flying Ethos radios because i introduced them to the system.
Happy flying Simon

Which are the new features which cause you problems in 1.5.X?

**Comment by bsongis-frsky** [comment]
It is difficult to explain why the conversion was wrong, I would need your previous data to know myself which mixes you were using before the conversion!

**Comment by T16-Pilot** [comment]
First of all, 1.5.X does work "fine" in it´s own way...

In my almost 3 years as an Ethos user i have never ever seen so many BUGS and rapid updates as we have seen in 1.5.X. We are now 3 months in with 1.5.X and we are already on version 9! (1.5.8)

For the record i am on 1.5.7 as 1.5.8 is not "stable" yet...

The major problem as I see it is that more or less every thing has changed to a more complicated way of doing things, aka the OTX way of doing things.

Small things like the timer in 1.5.X is now more complicated to setup than 1.4.17. It is not a problem when you figure out what to do. But the good old practice of "if it ain´t broke don´t fix it" has gone out the window on 1.5.X... most users just want a timer. Not a super mainframe computer running SpaceX Starship.

Same goes for Freemix, again now it is more complicated. Now we get a popup window when we want to change the weight. That means we need to press another 3 or 4 buttons to do the same thing as we did with one clik of a button before. And it looks like VARS is now a part of Freemix??

VARS is a new thing that most of us have never heard of, that we now need to try and learn and understand so that we can understand how the rest of the system reacts to this?

It is small things like that i find irritating.. it is like going back to a "Nintendo Gameboy" in the early 1990. A lot of clicking to go "no where". Or another way of putting it, try a Frsky Taranis X9D with OTX. That is a lot of clicking to get what you want... i was kind of hoping that way of working was history by now...?

I am not saying 1.5.X is not working, it is just that there are so many major changes in how to use the software compared to 1.4.17. It is scaring people of from investing in the system. 

1.4.17 was a nice platform that was easy to understand and use compared to 1.5.X.

And the fact that 1.5.X presents so many bugs is mind blowing to say the least..

The release of 1.5.X was way to early. 

Wait another 6 to 12 months and test the system before releasing it as an official release!


My 2 cent´s...

**Comment by bsongis-frsky** [comment]
Please have a look to the detailed release note of 1.5.8 and you will see that it also addresses issues from 1.4.X

You should also read a little bit about the requests which have been done, I don't think that any feature added to 1.5 has been done just for fun!

I am thinking right now about an option in Model / Edit. Some sort of checkbox Beginner / Expert. 

Choosing "Beginner" would make Ethos do the choices for you. For example the order of actions would be fixed (which fits to 95% users, including myself). The Vars would be hidden (80% people don't need them). The Timers would be simplified. Etc.

**Comment by spoke2570** [comment]
Didn't OTx have a little firmware feature checklist for it's installer? Maybe like that?

**Comment by T16-Pilot** [comment]
@bsongis bsongis-frsky wrote

"For example the order of actions would be fixed (which fits to 95% users, including myself). The Vars would be hidden (80% people don't need them). The Timers would be simplified. Etc."

What you are saying here is that 80% to 95% of the users of the system don´t need the complexity of the system?

That is exactly what i am trying to prove here...

So you are agreeing that Ethos is getting too complex for the major part of the users?

Why are we seeing all of these advance stuff in the system when 80-95% of the users don´t need it?

Are Frsky making a system for 5-15% of the users rather than making a system for the 80-95% of the users?

Who is paying the bill for the work the Ethos team is doing. Is it the 5-15% "Expert users" or the 80-95% "every day pilot" ? In my book it is simple math how is paying...

I know i am pushing it now, i am just trying to prove a point ;)

And no, i don´t think stuff are being added for fun NO! 

But the requests for a lot of the advanced stuff is coming from a very small community, and not the big user group of the system.

Just as an example, right now i am setting up a new model using 1.5.7. On my gain channel i need to set "weight" to 50% and offset to 50%. In 1.4.17 i had the "curve" on the screen when i did that. In 1.5.7 there is no "curve" when i go in to the new "menu" to adjust the weight and offset. Witch by the way is two different menu´s now, not as it was before. Before it was all on the same page and i could see on the output curve in the top right corner what i was doing.  Now i am working "blind". I got to step back to see if what i did was right or not.

As i wrote in my post above, small things that are irritating in 1.5.X

Don´t get me wrong, i do like the system. It is just getting a "little bit" over the top much for an every day pilot...

Best regards
Simon

**Comment by bsongis-frsky** [comment]
> What you are saying here is that 80% to 95% of the users of the system don´t need the complexity of the system?

But it is not because 90% people only need Rudder+Elevator+Throttle+Aileron that we should limit the system to these 4 functions.

**Comment by T16-Pilot** [comment]
> > What you are saying here is that 80% to 95% of the users of the system don´t need the complexity of the system?
> 
> But it is not because 90% people only need Rudder+Elevator+Throttle+Aileron that we should limit the system to these 4 functions.

Well lets be real now?

We where not talking about stripping it down to 4 channels.

What we are talking about is the fact that 1.5.X has taken a good working system (1.4.17) and turned it in to a more complicated user interface.

You admitted that 80-95% of the users don´t need the advanced functions. 

But once confronted with that fact you are responding that 90% only need 4 channels.

That was not the discussion...

Best regards
Simon

**Comment by bsongis-frsky** [comment]
My Rudder+Elevator+Throttle+Aileron example is just there to show you that your reasoning is wrong.

Waiting for feedback from more users, I am 100% sure that many will be happy when I will say I remove the Vars (or something else) to simplify the UI ;)

**Comment by T16-Pilot** [comment]
I am ok about being wrong, but i have been in this hobby since 1990... and i know a thing or two.

We are 10+ Ethos pilots in my club, mind you we are only like 25 active pilots. So there is a lot of Ethos in the club. Most of it because of me.

And when i here people talk about 1.5.X and people disliking the system, that scares me. Because there are people sitting in the fence thinking about converting to Ethos. And they hear the negative talk because things have change in the last updates. That is making potential pilots rethink...

So please, prove me wrong.

What i really want is the 1.4.17 mixer and timers back. The rest i could not care less about... oh and yes the wizard. When you select a gyro receiver and the wizard selects the channels, why on earth is the gain channel not defined correct with 0 to 100 and 50% offset?

Best regards
Simon

**Comment by williajgr** [comment]
There are way to many things that went from 1 screen to 3 screens to set things up.  I would go back to 1.4.x but there are bugs that need to be picked up.  I have many high end complicated jets that I fly with a lot of programming required.  Do you think I want to continue going through the programming to see what was messed up by the latest update?  If you do you are crazy.  If per chance one of my 20k jets goes in by your craziness there will be a law suit. Gliders don’t count only a few things to set up .  My jets have 20 channels of complicated setup.  And when everything is changed it is a real nightmare. Sent from my iPhoneOn May 12, 2024, at 11:12 AM, SBN ***@***.***> wrote:﻿
I am ok about being wrong, but i have been in this hobby since 1990... and i know a thing or two.
We are 10+ Ethos pilots in my club, mind you we are only like 25 active pilots. So there is a lot of Ethos in the club. Most of it because of me.
And when i here people talk about 1.5.X and people disliking the system, that scares me. Because there are people sitting in the fence thinking about converting to Ethos. And they hear the negative talk because things have change in the last updates. That is making potential pilots rethink...
So please, prove me wrong.
What i really want is the 1.4.17 mixer and timers back. The rest i could not care less about... oh and yes the wizard. When you select a gyro receiver and the wizard selects the channels, why on earth is the gain channel not defined correct with 0 to 100 and 50% offset?
Best regards
Simon

**Comment by T16-Pilot** [comment]
@williajgr  I was speaking for myself and what i see as a problem for me. I am also tired of going through every model every time i update my radio.

I do fly jet´s my self and i do not want to crash because of a programing error.

I think the Ethos team need to stop what they are doing and take a good long hard look at the hole system. And then start to fix what needs to be fixed before going forward with new functions.

Best regards
Simon

**Comment by williajgr** [comment]
You are making it like the futaba 9x which everyone hated.  Sent from my iPhoneOn May 12, 2024, at 4:55 PM, SBN ***@***.***> wrote:﻿
@williajgr  I was speaking for myself and what i see as a problem for me. I am also tired of going through every model every time i update my radio.
I do fly jet´s my self and i do not want to crash because of a programing error.
I think the Ethos team need to stop what they are doing and take a good long hard look at the hole system. And then start to fix what needs to be fixed before going forward with new functions.
Best regards
Simon

**Comment by robthomson** [comment]
I have to say that I think this is completely wrong.

The developers of ethos are responding and adding features that people are asking for.

Sure.. Some don't like all the extra functions.    But I guarantee that 70+% of users love the functions.

Do some of these features make it complicated?  Yes.   It's a very capable system.

Regardless of if it's Ethos or Futaba or Spektrum advice from any manufacturer will be to check everything after doing an update.  

If an individual does not  want to check everything solution is to update less often?. 

The firmware is in massive expansion mode at the moment and the feedback from real use is what will make it better.   If not up for the ride..  Wait 6 months.   Clearly there is a balance to be struck.  Not every new feature will be perfect on first go.  But like with all dev.. a few iterations on and the 'polish' will appear.  This polish only comes from user feedback and suggestions for improvements.   Simple criticisms and complaints do not ever move things forward or get results desired. 

All up.. If end users wanted a Spektrum..  Then they can use one. If they want everything dumbed down..  Great.   Other platforms are available.     No issue.  But why did they come to ethos in the first place?  They wanted more than the dumbed down platform gives.

For me..  The ethos devs are doing a bloody fantastic job.   The amount of attention to what end users are asking for and subsequent delivery of the requests are amazing.    A customer service at a level not seen from any other rc manufacturers.

**Comment by Dsrenger** [comment]
I totally agree to Rob.

People are often doing the mistake to think, that their own opinion stands for "90% of the users". If I need a feature, 90% of the users need it. If I dont need a feature, its to complicated and 90% of the users dont need it.

Both opinions usually get supported by "people at my field".

Are there people that prefer a less complex Ethos? Sure.
Are there people that want even more feature than 1.5? Sure.

The truth (and majority) is somewhere inbetween these 2 extremes. Thats the hard part for developers. Whatever they do, they will dissatisfy one of the sides.

For me, level of complexity is just perfect. But thats just my personal opinion.
I love Vars. I love that I can sperate the mixer logic from the Values used in the mixers. Makes the model so much easier to read. Even years after programming, because I know, when I have to adjust values, I just need to look in the Vars menu.

BUT, and that is the point, where I think the real problem is.

If somebody knows a little bit of programming, he understand very quickly where the advantages of Vars are.
A user that doesnt, has a hard time to understand.

The power users know, how to use the system.
But sometimes, it is hard the see, how Ethos "should" be used. The manual is very good (shoutout the Lothar) but it really cannot cover everything. Regardless of the fact, that people will not read it.

I struggled to understand how it is intended to program a model in the beginning.

I will pick an example from this issue here. It is just an example...

"I need different throws for camber left and right side"

This sentence shows, that the linkages are not symmetrical....Happens.... To make them "electronical equal" the intended way is to use the output menu on one channel and align the servos to each other.
Then, you just need one value in the mixer. It is the much cleaner way to program.

But, how should the user, that is not so deep in programming Ethos, know that? I explained that to several people after it got explained to me. If you think about it, it is clear, but is it obvious? Not really.

Can a manual cover that? Hardly. 

Perhaps a massive expansion of the Wizard could guide the user.  Choose plane configuration, let them adjusst the output curves in the wizard, set the numer of flight modes in wizard, RX battery warning setup in wizard ect. ect.

But where to start and where to end? It is a massive task and such a wizard would always just fit a part of the users. 
There are not just fixed wing modellers out there. And not everybody would program the same model in the same way. Conceptually it is a very hard task.

Long story short:
I can understand both sides.
The functionality of 1.5 is great and I dont want to miss a dime of it.
Is it complex. Not really, if you unterstood how Ethos intends to program a model.
But could Ethos in generall do a better job in explaining the "intended way of programming". I think so. Perhaps that would reduce the frustration of non power users.

But tbh...dont really know how to solve this without a massive amount of (concept)work.

I love 1.5 and the developers are doing a great job.

**Comment by Dsrenger** [comment]
Btw, We still dont have a model file to check, why the conversion went wrong.

**Comment by bsongis-frsky** [comment]
Back to the original issue and doing some guessing work which could be completely wrong:
- the user created a Var mix in 1.4
- this Var mix has been converted to a Var + FreeMix in 1.5
- it worked correctly but he needed to change some value and he couldn't find it (because it was inside the Vars page)
- he tried to simplify this mix by using only one FreeMix (which seems a good idea)
- he had some difficulties, the original screenshot seems very strange to me

**Comment by strgaltdel** [comment]
seems to be a plausible line of reasoning,
but without the ability to test the original 1.4.6 template, it's like looking into a crystal ball 

I would welcome a discussion without unnecessary emotion

- What was the goal of the "initial mixer" >> to achieve 3 fixed positions, if so which ones?
- how exactly it was realized in 1.4.x
- publish the original template so that anyone who is interested can replicate the bug or give support for the implementation in 1.5.x

imho:

even if there was a conversion problem, reaching three fixed positions in a mixer is a matter of a few minutes (definitely less than 5) and not worth hours of discussion
a precise description in the issue , more action is not required

if there is a user issue, RCG should be your friend, lots of helpful users. 

regarding complexity of ethos, most is said, 

- one cannot fullfill demands of all users with one single FW within the given ressources, 
- looking into git, there are a lot of enhancement request 
- and yes, a lot of users only need less then 40% of Ethos capabilities, but it's a mathemetical problem. 
- lets say 100% of users needs 30% of "core Ethos functions" 
- lets say the majority (e.g.80%) are using less then 35% of ethos, but they all do not use the same "less then 35%"
- lets say 4% are using "this 2%" of special ethos functions, next 4% are using "that 2%" of ethos functions.... so you end up with a an ethos which needs 30+80/4*2 = 70% of the actual ethos to fullfill the "needs of the base" , so we end up in a not so different FW from "todays ethos".
- add some functionality because of "not rational needs, but the competition commands it" and some percentages "for the freaks", there is not so much margin we can reduce

have said it on rcg, maybe one can focus a little bit more on usability, but reducing functionality is the wrong way.
if this would have been the solution, we all would have stayed with a 88' MC3030  


all In all
without template, no further steps possible, imho

**Comment by williajgr** [comment]
Ok here is the problem.   1 the devs don’t even know how it works.  They told me I could still set it up with settings 3 weights.  But you have to set up the middle switch position with a  offset not a weight.   2 something that took 1 screen now takes 4 screens.   3 I was at the field trying to change my flap setting, and couldn’t figure it out quickly as my programming had changed with out me knowing it.Sent from my iPhoneOn May 13, 2024, at 5:34 AM, Udo ***@***.***> wrote:﻿
seems to be a plausible line of reasoning,
but without the ability to test the original 1.4.6 template, it's like looking into a crystal ball
I would welcome a discussion without unnecessary emotion

What was the goal of the "initial mixer" >> to achieve 3 fixed positions, if so which ones?
how exactly it was realized in 1.4.x
publish the original template so that anyone who is interested can replicate the bug or give support for the implementation in 1.5.x

imho:
even if there was a conversion problem, reaching three fixed positions in a mixer is a matter of a few minutes (definitely less than 5) and not worth hours of discussion
a precise description in the issue , more action is not required
if there is a user issue, RCG should be your friend, lots of helpful users.
regarding complexity of ethos, most is said,

one cannot fullfill demands of all users with one single FW within the given ressources,
looking into git, there are a lot of enhancement request
and yes, a lot of users only need less then 40% of Ethos capabilities, but it's a mathemetical problem.
lets say 100% of users needs 30% of "core Ethos functions"
lets say the majority (e.g.80%) are using less then 35% of ethos, but they all do not use the same "less then 35%"
lets say 4% are using "this 2%" of special ethos functions, next 4% are using "that 2%" of ethos functions.... so you end up with a an ethos which needs 30+80/4*2 = 70% of the actual ethos to fullfill the "needs of the base" , so we end up in a not so different FW from "todays ethos".
add some functionality because of "not rational needs, but the compoetition commands it" and some percentages "for the freaks", there is not so much margin we can reduce

have said it on rcg, maybe one can focus a little bit more on usability, but reducing functionality is the wrong way.
if this would have been the soulution, we all woulöf have stayed with a 88' MC3030
all In all
without template, no further steps possible, imho

**Comment by Dsrenger** [comment]
Where is the problem just to attach your 1.4.17 model file, as requested several times, so we can check if something went wrong and perhaps improve the conversion. 

Or is the purpose of this here just to have a place to complain?

Note, I am not associated to FrSky in any way. Just to make sure my comment doesn't get associated with them. 

Perhaps you have valid points and things can get improved. But it's very hard to see without model file.

**Comment by bsongis-frsky** [comment]
> 1. the devs don’t even know how it works

How should I understand this point please?

It would be nice if you could explain clearly what you want to do! Then the devs can think of a solution for you, and see if it is + / - ~ complex in 1.5 compared to 1.4

**Comment by bsongis-frsky** [comment]
> 2. something that took 1 screen now takes 4 screens

I think this is wrong. Perhaps you need one offset and one weight (with perhaps multiple values)? It will depend of the source of the mix, you haven't told us what you have chosen.

But perhaps you need a slow or a curve? Then yes you will need to configure each action separately

**Comment by bsongis-frsky** [comment]
If you had a Var before and you need only a Free mix, then perhaps you need only one action which is Weight.
1) you will need Max as source
2) just keep one Weight, with 3 values for each position of your SE switch
